libraries/Microsoft.Bot.Builder.Prompts/BasePrompt.cs

```diff
@@ -7,7 +7,49 @@
 
 namespace Microsoft.Bot.Builder.Prompts
 {
+    public enum RecognitionStatus
```

Comment:
Should this maybe just be strings? 

That way we can evolve the list w/o breaking compat.

Reply:
I agree... I wouldn't specify any reasons. Let the consumer decide what they want their reasons to be.

---
In reply to: [171955473](https://github.com/Microsoft/botbuilder-dotnet/pull/232#discussion_r171955473) [](ancestors = 171955473)